benchexec/tools/symbiotic.py

```diff
@@ -25,19 +25,21 @@ class Tool(OldSymbiotic):
 
     REQUIRED_PATHS_7_0_0 = ["bin", "include", "properties", "lib", "llvm-8.0.1"]
 
-    def executable(self):
+    def executable(self, tool_locator):
         """
         Find the path to the executable file that will get executed.
         This method always needs to be overridden,
         and most implementations will look similar to this one.
         The path returned should be relative to the current directory.
         """
-        exe = util.find_executable("bin/symbiotic", exitOnError=False)
-        if exe:
-            return exe
-        else:
+        # This is used in the function _version_newer_than.
+        # I don't know a better way to do it.
```

Comment:
Such a comment should not be part of the code. If you have questions, please mark the PR as draft and add the questions to the PR discussion instead.